src/services/analytics/events/recovery.ts

```diff
@@ -0,0 +1,46 @@
+import { EventType } from '@/services/analytics'
+
+// These are used for the generic stepper flow events (Next, Back)
+export const SETUP_RECOVERY_CATEGORY = 'setup-recovery'
+export const START_RECOVERY_CATEGORY = 'start-recovery'
+export const CANCEL_RECOVERY_CATEGORY = 'cancel-recovery'
+
+const RECOVERY_CATEGORY = 'recovery'
+
+export const RECOVERY_EVENTS = {
+  SETUP_RECOVERY: {
+    action: 'Set up recovery',
+    category: RECOVERY_CATEGORY,
+    event: EventType.CLICK,
+  },
+  START_RECOVERY: {
+    action: 'Start recovery',
+    category: RECOVERY_CATEGORY,
+    event: EventType.CLICK,
+  },
+  CANCEL_RECOVERY: {
+    action: 'Cancel recovery',
+    category: RECOVERY_CATEGORY,
+    event: EventType.CLICK,
+  },
+  SHOW_ADVANCED: {
+    action: 'Show advanced recovery settings',
+    category: RECOVERY_CATEGORY,
+    event: EventType.CLICK,
+  },
+  DISMISS_PROPOSAL_CARD: {
+    action: 'Do it later',
+    category: RECOVERY_CATEGORY,
+    event: EventType.CLICK,
+  },
+  LEARN_MORE: {
+    action: 'Learn more',
+    category: RECOVERY_CATEGORY,
+    event: EventType.CLICK,
+  },
+  GO_BACK: {
+    action: 'Go back on cancel',
```

Comment:
The other evenrs in this file have "recovery" in the name, but these don't and they can be ambigous unless the category is used as well.

Reply:
We can make these actions longer and say `Learn more about recovery` etc.